examples/basic/pd-micro-service-cluster.yaml

```diff
@@ -7,7 +7,10 @@ metadata:
   name: basic
   namespace: pingcap
 spec:
-  version: v7.1.1
+  startScriptVersion: v2
+  startScriptV2FeatureFlags:
+    - WaitForDnsNameIpMatch
+  version: v7.5.0
```

Comment:
If we use PD ms in start script version v1, it still should work well (but may restart sometimes), right?

Reply:
changed to PD&PDMS use start script v2 when enabled PDMS